server/src/services/handover.service.ts

```diff
@@ -0,0 +1,51 @@
+import { UpdateResult } from "typeorm";
+import { errorMsg, errorStatusCode } from "../bases/errorTypes";
+import ErrorHandler from "../controller/error.controller";
+import { Handover } from "../entity/handover";
+import { HandoverRepository } from "../repository/handover.repository";
+
+export class HandoverService {
+  constructor(private readonly repository: HandoverRepository) {}
+
+  async getAll(): Promise<Handover[]> {
+    const handover = await this.repository.getAll();
+    return handover;
+  }
+
+  async getById(id: number): Promise<Handover | undefined> {
+    const handover = await this.repository.getById(id);
+    return handover;
+  }
+
+  async checkExistByID(id: number): Promise<boolean> {
```

Comment:
whats difference between this and getByID?

Reply:
I deleted